src/test/components/__snapshots__/ProfileCallTreeView.test.js.snap

```diff
@@ -5752,11 +5763,12 @@ exports[`calltree/ProfileCallTreeView renders call tree with some search strings
       data-index="0"
       title="Complete \\"Empty\\""
     >
-      <span
+      <button
         class="filterNavigatorBarItemContent"
+        type="button"
       >
         Complete "Empty"
-      </span>
+      </button>
     </li>
   </ol>
   <div
```

Comment:
I see there's no snapshot with a filter bar that has more than 1 element (eg: display the top bar where we have a committed range, or the transform bar with a transform). Do you think you can add one?